Bottom: 931edab48525dc659b78519fb0cc63fee9469fab
Top:    f1b7e21968beea0d56af76b4f20cd640b851e0b5
Author: Peter Clifton <pcjc2@cam.ac.uk>
Date:   2011-04-20 17:39:37 +0100

Refresh of tidy-up-pad-drawing-routines

---

diff --git a/src/crosshair.c b/src/crosshair.c
index a9ed879..a3d5da7 100644
--- a/src/crosshair.c
+++ b/src/crosshair.c
@@ -255,16 +255,16 @@ XORDrawElement (ElementTypePtr Element, LocationType DX, LocationType DY)
   /* pads */
   PAD_LOOP (Element);
   {
-    /* Make a copy of the pad structure, moved to the correct position */
-    PadType moved_pad = *pad;
-    moved_pad.Point1.X += DX;
-    moved_pad.Point1.Y += DY;
-    moved_pad.Point2.X += DX;
-    moved_pad.Point2.Y += DY;
-
     if (PCB->InvisibleObjectsOn ||
         (TEST_FLAG (ONSOLDERFLAG, &moved_pad) != 0) == Settings.ShowSolderSide)
-      gui->thindraw_pcb_pad (Crosshair.GC, &moved_pad, false, false);
+      {
+        /* Make a copy of the pad structure, moved to the correct position */
+        PadType moved_pad = *pad;
+        moved_pad.Point1.X += DX; moved_pad.Point1.Y += DY;
+        moved_pad.Point2.X += DX; moved_pad.Point2.Y += DY;
+
+        gui->thindraw_pcb_pad (Crosshair.GC, &moved_pad, false, false);
+      }
   }
   END_LOOP;
   /* mark */
